hammer_time/hammer_time.py

```diff
@@ -52,6 +78,19 @@ def perform(client, host_id):
         remove_and_wait(client, sorted(new_cont))
 
 
+class KillMongoDAction:
+    """Action to kill mongod."""
+
+    def generate_parameters(client):
+        return {'machine_id': choose_machine(client.get_controller_client())}
+
+    def perform(client, machine_id):
+        ctrl_client = client.get_controller_client()
+        ctrl_client.juju('ssh', (machine_id, 'sudo', 'pkill', 'mongod'))
+        # The impact of killing mongod seems to be delayed.
+        time.sleep(5)
```

Comment:
Would polling for a pid change or the absence of a pid be more reliable and informative?